### PR TITLE
Visitor can click on datapoint and see Post

### PR DESCRIPTION
**Pull request by raptorf1**
# [PT Story](https://www.pivotaltracker.com/story/show/166572478)

### Changes proposed in this PR
* Acceptance test for visitor being able to click a dot in the map and see the correct information
* Visitor can click on a dot in the map and see a pop-up with image, caption, formatted date & time and formatted address
* The pop-up has a coloured frame depending on the category of the `Entry`
* Pop-up closes when user clicks on the map in the background

### What we have learned
* Async-await in action

### Screenshots

Screenshot from 2019-06-21 00-38-33

### CODE REVIEW

**Review by leiter007 — COMMENTED** [review]
Looks AMAZING! Super good work! Pulled it down and works really good. Only question is if we should style for like 3 hours max on this, and then merge it..?

**Comment by raptorf1** [comment]
> Looks AMAZING! Super good work! Pulled it down and works really good. Only question is if we should style for like 3 hours max on this, and then merge it..?

I think that's a good limit. I would say that the pop-up will already have colours from the photo, so the background should remain white and the information font colour black. We could make the pop-up appear top-centre and not restrict the image height. No problem in scrolling down I think. That's why you sign-up, to see the whole pictures. Of course the image should be inside the pop-up.

File: src/Components/EntryPopup.jsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      },
+      error => {
+        console.error(error);
+      }
+    )
+  }
+
+  executeOnClick(isExpanded) {
+    console.log(isExpanded);
+  }
```
Comment (raptorf1) [inline src/Components/EntryPopup.jsx]: do we need this now that `react-show-more` is removed?

Comment (zanenkn) [reply src/Components/EntryPopup.jsx]: nope. take it away.

**Pull request merged**